web/src/app/util/TelTextField.js

```diff
@@ -0,0 +1,99 @@
+import React, { useEffect, useState } from 'react'
+import TextField from '@material-ui/core/TextField'
+import { Check, Close } from '@material-ui/icons'
+import _ from 'lodash-es'
+import InputAdornment from '@material-ui/core/InputAdornment'
+import { makeStyles } from '@material-ui/core'
+import { DEBOUNCE_DELAY } from '../config'
+
+import gql from 'graphql-tag'
+import { useQuery } from '@apollo/react-hooks'
+
+const isValidNumber = gql`
+  query($number: String!) {
+    phoneNumberInfo(number: $number) {
+      id
+      valid
+    }
+  }
+`
+
+const useStyles = makeStyles({
+  valid: {
+    fill: 'green',
+  },
+  invalid: {
+    fill: 'red',
+  },
+})
+
+export default function TelTextField({ ...props }) {
+  const classes = useStyles()
+  const [phoneNumber, setPhoneNumber] = useState(null)
```

Comment:
Start this as an empty string
```suggestion
  const [phoneNumber, setPhoneNumber] = useState('')
```